### PR TITLE
Add leave guidance

### PR DESCRIPTION
**Pull request by ctro**
Adding basic leave guidance

### CODE REVIEW

**Review by sharms — APPROVED** [review]
LGTM!

**Review by pauldoomgov — APPROVED** [review]
LGTM

File: _articles/leave-guidance.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+## Login.gov Leave Guidance
+
+Please follow [leave guidance in the TTS Handbook](https://handbook.tts.gsa.gov/travel-and-leave/leave/)
+
+Additionally, please do the following
+
+- Give at least one-sprint, but preferably two-sprint notice if you are going to be out for more than a couple days
+- Put your OOO time in the Shared Login.gov Calendar and decline meetings
+- List your back-ups in GMail auto-replies and Slack profile
```
Comment (aduth) [inline _articles/leave-guidance.md:15]: A question from ignorance: Who are my back-ups? Does this apply to me?

Comment (ctro) [reply _articles/leave-guidance.md:15]: If you don't know who they are, then you probably don't have any.  
If there's something you're responsible for and you'll be away for a week, consider talking to someone about being your back-up.   Maybe "Any questions about  LGDesignSystem should be directed to Nick", etc.

How could we write that to be a little more clear?

Comment (aduth) [reply _articles/leave-guidance.md:15]: That makes a lot of sense to me just as you've written there. Would it be too lengthy to include in-line? Maybe as a sub-bullet point?

```suggestion
- List your back-ups in GMail auto-replies and Slack profile. If you don't know who they are, then you probably don't have any. If there's something you're responsible for and you'll be away for a week, consider talking to someone about being your back-up.
```

**Pull request merged**